src/clients/discord.ts

```diff
@@ -92,38 +91,38 @@ function registerDiscord(): void
         for(const msg of chunkedTwitchMessages)
             bridge.twitchMessageSearchCache[msg] = listNode;
 
-        let currIndex = 0;
-            const recursiveSend = ()=>
-            {
-                if(currIndex < chunkedTwitchMessages.length)
-                bridge.twitchClient!.say(configFile.T2D_CHANNELS[0], chunkedTwitchMessages[currIndex]).then(()=>
+        let currIndex: number = 0;
+        const recursiveSend = (): void =>
+        {
+            if(currIndex < chunkedTwitchMessages.length)
+                bridge.twitch.authChatClient!.say(configFile.T2D_CHANNELS[0], chunkedTwitchMessages[currIndex]).then(() =>
                 {
                     currIndex++;
                     //tmijs does this in their own version of sending multiple messages... therefore we must also follow this jank method
-                    setTimeout(()=>recursiveSend(), 250);
+                    setTimeout(() => recursiveSend(), 250);
                 });
-            }
+        };
 
-            recursiveSend();
+        recursiveSend();
 
         //Count upwards and delete the oldest message if need be
         manageMsgCache();
     });
 
+    /**
+     * Current bugged, will always throw an error due to us never getting the twitchObj bound to the linkedListNode.
```

Comment:
bruh

Reply:
That actually works! It didn't on the starting commits and I forgot to remove that comment (: